server/src/logic.js

```diff
@@ -60,8 +61,11 @@ export const userHandler = {
     return user.getEvents();
   },
   schedules() {
-    // TODO: implement this
-    return [];
+    return Schedule.findAll({
+      include: [
+        { model: Group, required: true },
```

Comment:
this doesn't actually filter by user though. So every time we resolve this for any user we will return all schedules from all groups (it's not even limited by organisation!)
If you're just looking to stuff data in the API this is fine but lets make a note of it with a comment.

Reply:
You sure? in my testing it seems to only return things that user had a relation to. I created another set of groups that this user was not in and they are not returned